source/renderer/app/components/loading/syncing-connecting/SyncingProgress/SyncingProgress.tsx

```diff
@@ -40,6 +42,17 @@ const makePercentageCellStyles = (loaded: boolean) =>
     [styles.faded]: loaded,
   });
 
+const getSafePercentage = (value: number): string => {
+  try {
+    return new BigNumber(value).toFixed(2).toString();
+  } catch (error) {
+    logger.error('SyncingProgress::Percentage::Error parsing sync percentage', {
+      error,
+    });
+    return '-';
```

Comment:
@danielmain is this `-` a requirement or we can return e.g. `0%` ?

Reply:
I am uncertain if onParsingError we show 0% percentage sync. People will think they have to wait if they see 0% where there is actually an error.

Should we show "Error parsing" ?

Reply:
hmmm, yeah. Maybe we can keep it as is and show `-`. It is better than showing the error that we are not able to parse the data